astropy_timeseries/binned.py

```diff
@@ -111,6 +116,7 @@ def __init__(self, data=None, time_bin_start=None, time_bin_end=None,
                     time_bin_end = times
                 time_bin_size = (time_bin_end - time_bin_start).sec * u.s
             elif time_bin_size is None:
+                # I am not sure this is triggerable.
```

Comment:
@nabobalis - I can trigger it with:

```python
In [1]: from astropy_timeseries import BinnedTimeSeries

In [2]: BinnedTimeSeries(time_bin_start=['2016-03-22T12:30:31',
   ...:                                    '2016-03-22T12:30:34',
   ...:                                    '2016-03-22T12:30:37',
   ...:                                    '2016-03-22T12:30:39'])
   ...:                                    
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-2-2183cf3da47e> in <module>()
      2                                    '2016-03-22T12:30:34',
      3                                    '2016-03-22T12:30:37',
----> 4                                    '2016-03-22T12:30:39'])
      5 

~/miniconda3/envs/dev/lib/python3.7/site-packages/astropy_timeseries/binned.py in __init__(self, data, time_bin_start, time_bin_end, time_bin_size, n_bins, **kwargs)
     57 
     58         if time_bin_end is None and time_bin_size is None:
---> 59             raise TypeError("Either 'time_bin_size' or 'time_bin_end' should be specified")
     60 
     61         if not isinstance(time_bin_start, Time):

TypeError: Either 'time_bin_size' or 'time_bin_end' should be specified
```

Reply:
That seems to be the error from line 59, which I believe prevents this error line from being triggered.

Reply:
@nabobalis - you are correct, can you remove this if branch?

Reply:
I removed just the elif.